pkg/cmd/cobra/cobra.go

```diff
@@ -320,9 +320,6 @@ func GetTraceeRunner(c *cobra.Command, version string) (cmd.Runner, error) {
 	runner.Printer = p
 	runner.InstallPath = traceeInstallPath
 
-	// parse arguments must be enabled if the rule engine is part of the pipeline
-	runner.TraceeConfig.Output.ParseArguments = true
```

Comment:
It seems that kernel tests require it on: https://github.com/aquasecurity/tracee/actions/runs/8555281070/job/23442355573?pr=3960#step:7:74